BukkyOmo/Questioner
# #163748462 Implement FETCH API to get all meetups page.

#### What does this PR do?
This PR:
* Implement Fetching API for get all meetups endpoint.

#### Description of Task to be completed
* Create a js file for signup.
* Include the get all meetups page heroku page url.
* Create variables for each field that is required.
* Create an event listener for the button to be clicked. 
* Create your fetch which should include method of http request, body and the headers.

#### How should this be manually tested?
* Clone this repository
* Check out to <ft-implement-fetch-API-for-get-all-meetups-page-#163748462> branch
* Check meetups.html page via browser.

#### Any background context you want to provide?
No

#### What are the relevant pivotal tracker stories? (if applicable)
#163748462
